### ISSUE TITLE
JSON files not checked after new VS Code update

### ISSUE THREAD

**Issue by AlexeiSuleac** [Jason-Rev/vscode-spell-checker#41]
JSON files are not being checked any more after 1.8 VS Code release.
Settings in _cSpell.json_
`"enabledLanguageIds": [
        "csharp",
        "go",
        "json",
        "javascript",
        "javascriptreact",
        "markdown",
        "php",
        "plaintext",
        "text",
        "typescript",
        "typescriptreact",
        "yml",
        "feature"
    ]`

**Comment by Jason3S** [comment]
Thank you.  I think VS Code changed the way they load the _settings_ files.  See: [Release Notes: November 2016](https://code.visualstudio.com/updates#_settings)

As a workaround, you can change your user settings and add it there.  I haven't had time yet to find out what the real change was.  

It will look something like this in your user or project settings file:

```javascript
"cSpell.enabledLanguageIds": [ 
    "csharp",
    "go",
    "json",
    "javascript",
    "javascriptreact",
    "markdown",
    "php",
    "plaintext",
    "text",
    "typescript",
    "typescriptreact",
    "yml",
    "feature" 
]
```

**Comment by AlexeiSuleac** [comment]
Cool. Thank you....

**Comment by Jason3S** [comment]
This should be fixed with 0.14.0
